tuts/create-your-first-substrate-chain/v2.0.0-rc2/interact.md

```diff
@@ -61,41 +61,72 @@ $ ./target/release/node-template --dev
 
-In your web browser, navigate to
-[https://polkadot.js.org/apps](https://polkadot.js.org/apps/#/settings?rpc=ws://127.0.0.1:9944).
+To get started with the Front-End Template, clone its repository and
+[follow the simple steps to run it locally](https://github.com/substrate-developer-hub/substrate-front-end-template#using-the-template).
 
-On the `Settings` tab ensure that you are connected to a `Local Node` or `ws://127.0.0.1:9944`.
+## Interact
 
-> Some browsers, notably Firefox, will not connect to a local node from an https website. An easy
-> work around is to try another browser, like Chromium. Another option is to
-> [host this interface locally](https://github.com/polkadot-js/apps#development).
+Once the Front-End Template is running and loaded in your browser at
+[http://localhost:8000/](http://localhost:8000/), take a moment to explore its components. At the
+top, you will find lots of helpful information about the chain to which you're connected as well as
+an account selector that will let you control the account you use to perform on-chain operations.
 
-## Interact
+![Chain Data & Account Selector](../assets/chain-data.png)
 
-Select the **Accounts** tab, you will see test accounts that you have access to. Some, like Alice
-and Bob, already have funds!
+There is also a table that lists
+[the well-known test accounts](/kb/integrate/subkey#well-known-keys) that you have access to. Some,
+like Alice and Bob, already have funds!
 
-![Apps UI with pre-funded accounts](../assets/apps-prefunded.png)
+![Account Table](../assets/accts-prefunded.png)
 
-You can try to transfer some funds from Alice to Charlie by clicking the "send" button.
+Beneath the account table there is a Transfer component that you can use to transfer some funds from
+one account to another. Take note of the info box that describes the precision used by the Front-End
+Template; you should transfer at least `1000000000000` to make it easy for you to observe the
+changes you're making.
 
 ![Balance Transfer](../assets/apps-transfer.png)
 
-If everything went successfully, you should see some popup notifications claiming "Extrinsic
-Success", and of course Charlie's balance will increase.
+Notice that the table of accounts is dynamic and that the account balances are updated as soon as
+the transfer is processed.
+
+### Runtime Metadata
+
+The Front-End Template exposes many helpful features and you should explore all of them while you're
+connected to a local development node. One good way to get started is by clicking the "Show
+Metadata" button at the top of the template page and reviewing
+[the metadata that your runtime exposes](https://www.substrate.io/kb/runtime/metadata).
+
+![Metadata JSON](../assets/metadata.png)
+
+### Pallet Interactor & Events
+
+You can use the runtime metadata to discover a runtime's capabilities. The Front-End Template
+provides a helpful Pallet Interactor component that provides several mechanisms for interacting with
```

Comment:
I guess it's a real word huh? TIL!

Reply:
I do not believe this is a "real word" :+1:

Reply:
Maybe we can make it so. ;)